src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,40 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
```

Comment:
It is an ugly fix, as I've stated.
I had to add the second if to make our functional tests work, that's the only reason it's there.

There are probably **many** cases where this hack doesn't work or even destroys a functionality that would have worked without the fix.

This is merely a 'proposition' for us to either accept or reject.
If rejected, then we wont have the desired functionality until the real solution is implemented.

However, adding the constaint of 'only one incoming CE' seems like a good idea to me.